docs/CONTRIBUTING.md

```diff
@@ -186,6 +186,36 @@ Once the PR is merged into `master` a build is triggered on [SemaphoreCI](https:
 
 > see [build configuration](https://semaphoreci.com/lmem/extension/settings)
 
+### Profiles
+
+#### Staging & proding
+
+`staging`and `proding` environment are automatically deployed on __Clever Cloud__ on a new push on the `master` branch:
+
+- https://staging-profiles.dismoi.io/
+- https://proding-profiles.dismoi.io/
+
+#### Production
+
+DisMoi `production` profiles app is available at https://www.dismoi.io/eclaireurs/.
```

Comment:
Prod version is first in profiles.dismoi.io (corvée cloud) and then Linked in dismoi.io

Reply:
> corvée cloud

:rofl: 

That's what I tried to express in the second paragraph, but apparently not clear enough. What's your suggestion?

Reply:
Clear enough for me (about Corvée Cloud :-) )

Reply:
corvée cloud :-) thx to smartphone